ci/do_ci.sh

```diff
@@ -34,7 +34,7 @@ function do_test() {
 }
 
 function do_clang_tidy() {
-    ci/run_clang_tidy.sh
+    ci/run_clang_tidy.sh || true
```

Comment:
Can we add a comment here pointing to the issue about a long term fix?

Reply:
Done in 0790741